include/arch/arm/aarch64/cpu.h

```diff
@@ -171,6 +171,7 @@
 #define MPIDR_AFFLVL(mpidr, aff_level) \
 		(((mpidr) >> MPIDR_AFF##aff_level##_SHIFT) & MPIDR_AFFLVL_MASK)
 
-#define GET_MPIDR()		read_sysreg(mpidr_el1)
+#define __get_MPIDR()		read_sysreg(mpidr_el1)
```

Comment:
@andyross   >> Any reason for the oddball capitalization?
CMSIS (M/R/ v7A)  provides these definitions.  CMSIS for armv8-a is not available.
We will try to keep the needed definitions signatures same as CMSIS so that where they are used
they can be Cortex-A/M/R/V8A  agnostic.

Reply:
Unless we plan to switch the entire AArch64 CPU interface to CMSIS-style, IMHO, this should stay `GET_MPIDR`, and its users should do `#ifdef`.

This is one of the reasons why I suggested defining AArch64 symbols in CMSIS-style earlier.

Reply:
Ideal is to have CMSIS like defines but Cortex-A Aarch64 CMSIS is not planned as per what I learned from arm tools team.
If more products use Cortex-A aarch64 as a high-end micro controller things can change (unlikely).

It also depends on where we need the abstraction.
ex: __DSB()  we have it defined as CMSIS. but really this should be abstracted at OS level then down the layer if its from CMSIS or some arch define would not matter.  Treating Aarch64 as a non-CMSIS port made sense so far.
But I am not against making all the defines CMSIS style, just that lack of time at the moment.

I can fallback to own define and add #ifdef at interrupt test routine. Because later point the gic_send_sgi()  itself needs an abstraction with cpu logical map if ever SMP support is added.

Reply:
Use non-CMSIS for ARM64.